.github/workflows/build.yaml

```diff
@@ -0,0 +1,13 @@
+on: [push, pull_request]
+name: Build
+jobs:
+    travis-ci:
+        name: Travis CI
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@master
+            - name: Travis CI
+              uses: travis-ci/actions@master
+              secrets = [
+                  ${{ secrets.TRAVIS_TOKEN }},
+              ]
```

Comment:
```suggestion

```

Reply:
`secrets` replace by `secret`

Reply:
Not sure it’s right, because I don’t know if this is valid yaml anyway:

secret = [...]

Reply:
Github error say that error. I saw many article from github action with this syntax.

Reply:
If you want, tonight i try github action with my personal project affiliate travis ci for test.

Reply:
Yes please

Reply:
With my [build.yaml](With my build.yaml i have got access denied i need generate a token. But for this repo, i think is good.) i have got access denied i need generate a token. But for this repo, i think is good.

I think an article for create an access token for travis : [Link](https://medium.com/@JoshuaTheMiller/retrieving-your-travis-ci-api-access-token-bc706b2b625a)

I search today for find a solution